tools/registrar-scraper/src/instructor-scraper/index.ts

```diff
@@ -26,7 +26,7 @@ type InstructorsData = {
 };
 
 type InstructorsInfo = {
-  [ucinetid: string]: Instructor;
+  [ucinetid: string]: Omit<Instructor, "courses">;
```

Comment:
I prefer using records if there's no information to preserve.
```suggestion
  Record<string, Omit<Instructor, "courses">>
```
 The main reason I use the mapped is to preserve interface information.

```ts
type MyType = {
  /** JSDoc for property a */
  a: string,
  /** JSDoc for property b*/
  b: string,
}

type DerivedType = Record<keyof MyType, any>
// Lost all of the JSDoc info.
type DerivedTypeA = DerivedType["a"]

type BetterDerived = { [k in keyof MyType]: MyType[k]}
// The JSDoc is preserved.
type BetterDerivedA = BetterDerived["a"]

If the key itself is a string, then there isn't any JSDoc information to preserve anyways.
```